srcpkgs/simple-obfs/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'simple-obfs'
+pkgname=simple-obfs
+version=0.0.5
+revision=1
+build_style=gnu-configure
+hostmakedepends="git automake libtool pkg-config asciidoc xmlto"
+makedepends="libev-devel"
+short_desc="A simple obfusacting tool designed as plugin server of shadowsocks"
+maintainer="whoami <whoami@systemli.org>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/shadowsocks/simple-obfs"
+
+do_fetch() {
```

Comment:
No, use a tagged release on distfiles=

git packages are very **bad**

Reply:
I can't use distfiles, because it requires a git submodule `libcork`, but it's not present in archive file https://github.com/shadowsocks/simple-obfs